.github/workflows/release.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Release
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CI: true
+  RUBY_VERSION: '2.6'
+  JAVA_VERSION: adopt@v8
+
+jobs:
+  publish-microsite:
+    name: Publish microsite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v5
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Install Jekyll
+        run: gem install jekyll -v 4
+      - name: Build
+        run: sbt docs/makeMicrosite
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are these secrets local to this repository?

Reply:
It is automatically provided when using GitHub Actions, we do not need to generate it or set it up, it is already handled by GitHub Actions.

Docs: https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret